src/lib/plugin_apis/lvm.api

```diff
@@ -307,6 +309,9 @@ typedef struct BDLVMLVdata {
     guint64 metadata_percent;
     guint64 copy_percent;
     gchar **lv_tags;
+    gchar *health_status;
+    gchar **devices;
```

Comment:
Btw. `lvs -o devices` also doesn't contain metadata which can be located on different PVs. I think we'll need something way more complicated to this unfortunately.

Reply:
> I think this needs some explanation what it contains.

Ultimately, it is meant to show which PVs contribute space for a given LV.  My plan right now is to let the user select which PVs to use when creating a LV, and then it is only approriate to show what they are also afterwards.

I realize only now that I haven't linked this PR back to the full story, sorry.  I'll correct that.

> If the idea here is to get "list of PVs this LV is allocated on", we should provide a better results than lvs -o+devices does.

Yes, and currently this is produced in UDisks: https://github.com/storaged-project/udisks/pull/969/files#diff-6d18017b318477b8f7b0121bae3305c5c35f3ebeed6da974725a51a6a42f2961R229

If this should happen in libblockdev instead, I am all for it.

Reply:
> Btw. `lvs -o devices` also doesn't contain metadata which can be located on different PVs so we'd need to use both `devices` and `metadata_devices`. I think we'll need something way more complicated to this unfortunately.

Interesting. Thanks!

Reply:
Also, `devices` and `metadata_devices` are actually `Linear Volume Segment Fields`, and if we ask for them with `-o` we get one line of output for each output for each segment of a LV, not the expected single one.  The same is true for `segtype`, which is already in the list...

Reply:
Or we omit `devices` etc completely here.  It is only used for informational purposes in Cockpit, and not really required to create and repair RAID LVs.

Reply:
> > It would be nice to have a full "map" of the segments with ranges in libblockdev.
> 
> Yes, I see. But doesn't this contradict "We should definitely move [pv collection] to libblockdev."? Either libblockdev gives a list of segments, or it gives a pre-cooked list of PVs, no? Or do you want it to do both?
> 
> Would it make sense to exclude `devices` from this PR and move this whole topic to a separate one? From Cockpit's point of view `devices` is only "nice to have".

If it's just a nice-to-have, it would be better to implement this separately. I was thinking about having both segments and PVs, probably as a separate struct. We can create a separate RFE for this a discuss it more later.

Reply:
Roger.